src/Processors/QtiV2/In/Interactions/ChoiceInteractionMapper.php

```diff
@@ -90,4 +97,47 @@ private function buildOptions(SimpleChoiceCollection $simpleChoices)
         }
         return $options;
     }
+
+    private function buildFeedbackMetadata(SimpleChoiceCollection $simpleChoices)
+    {
+        /* @var $choice SimpleChoice */
+        $metadata = [];
+        foreach ($simpleChoices as $key => $choice) {
+            $flow = $choice->getContent();
+            $class = new \ReflectionClass(get_class($flow));
```

Comment:
Can you move this line into the `if` block below? It is never used outside of the `if`, so we can avoid creating it until we need it.

Reply:
Done